RELEASE_NOTES.md

```diff
@@ -1,42 +1,263 @@
+
+For a complete description of the capabilities of lazy attribute evaluation see https://github.com/chef/chef/pull/10861
+
+#### Custom Resource Property Defaults
+
+Chef Infra Client's handling of default property values in Custom Resources has been improved to avoid potential Ruby errors. These values are now duplicated internally allowing them to be modified by the user in their recipes without potentially receiving fatal frozen value modification errors.
+
+#### effortless? helper
+
+A new `effortless?` helper identifies if a system is running Chef Infra Client using the Effortless Pattern.
+
+#### reboot_pending? Improvements
+
+The `reboot_pending?` helper now works on all Debian based platforms instead of just Ubuntu.
 
 ### Resource Improvements
 
-- apt_package allow_downgrades now functions as expected, but also raises on invalid versions - https://github.com/chef/chef/pull/10993
-- Use shell redirection in chef_client_cron when append_log_file is true - https://github.com/chef/chef/pull/11124
-- Improve idempotency debug logging in resources - https://github.com/chef/chef/pull/11149
-- Resolve potential failures in chef_client_launchd and macosx_service - https://github.com/chef/chef/pull/11154
-- Improved performance in systemd_unit resource - https://github.com/chef/chef/pull/10925
-- gem resource: assume rubygems 1.8+ now: https://github.com/chef/chef/pull/10379
-- execute: Add login property - https://github.com/chef/chef/pull/11201
+#### Logging Improvements
+
+A large number of resources have seen improvements to the logging available in the `debug` log level providing better information for troubleshooting Chef Infra Client execution. Thanks for this improvement [@jaymzh](https://github.com/jaymzh)!
+
+#### apt_package
+
+The `apt_package` resource now properly handles downgrading package versions. Please note that full versions must be provided in the `version` property and invalid version strings will now raise and error. Thanks for this improvement [@jaymzh](https://github.com/jaymzh)!
+
+#### chef_client_launchd / macosx_service
+
+The `chef_client_launchd` and `macosx_service` resources have been updated to use the full path to the `launchctl` command. This avoids failures running these resources with incorrect PATH environment variables. Thanks for this improvement [@krackajak](https://github.com/krackajak)!
+
+#### systemd_unit
+
+The `systemd_unit` resource has been improved to only shell out a single time to determine the state of the systemd unit. This optimization can result in significant performance improvements when using large numbers of `systemd_unit` resources. Thanks [@joshuamiller01](https://github.com/joshuamiller01)!
+
+#### execute
+
+The `execute` resource includes a new `login` property allowing you to run commands with a login shell. This is useful to ensure you have all potential environment variables defined in the user's shell.
 
 ### Ohai
```

Comment:
the cpu plugin was nearly completley rewritten by @ramereth  and includes cpu vuln info and supports a bunch of other new data that I'm sure he can tell you about.

Reply:
Added